src/app/workflow-app/workflow-app.component.html

```diff
@@ -34,6 +34,52 @@
   ></workflow-list>
   <workflow-warnings #warnings [hidden]="activeMenuItem !== 'Warnings'"></workflow-warnings>
   <zosmf-server-config #zosmfserverconfig (configured)="onConfigured($event)" [hidden]="activeMenuItem !== 'Configuration'"></zosmf-server-config>
+  <workflow-popup
```

Comment:
Why do you need to make and use a workflow-popup component, rather than just use zlux-popup-window?

Maybe you have a good reason, but it requires an explanation.

Reply:
I can't change the UI of the ZluxPopupWindow, at least until that ticket that I put up gets solved. We can use an "@Input" binding and put in some parameters... and this will work for our width issue, but ultimately this doesn't solve the problem and it may be that we would want to change the UI of that dialog for something specific to Workflow. Maybe not much in this situation, since we want to have UI consistent with the other Zowe widgets, but inputting a dozen parameters may not be as efficient as creating a dedicated component such as Workflow has done already with the workflow-list, workflow-notification, workflow-step components, etc. After all, if we are creating an element for Workflow (a config and save changes popup) that is specific to the plugin, don't we want full control over that component? What do you think?